src/get-started/java-binding.md

```diff
@@ -555,31 +560,31 @@ For using the library just include the dependency in your `pom.xml`:
 - [Rust instruction](create-service.md)
 - [Java Binding App tutorial][app-tutorial]
 
-[abstractservice]: https://exonum.com/doc/api/java-binding-core/0.4/com/exonum/binding/service/AbstractService.html
-[apicontrollertest]: https://github.com/exonum/exonum-java-binding/blob/v0.4/exonum-java-binding/cryptocurrency-demo/src/test/java/com/exonum/binding/cryptocurrency/ApiControllerTest.java
-[app-tutorial]: https://github.com/exonum/exonum-java-binding/blob/v0.4/exonum-java-binding/core/rust/ejb-app/TUTORIAL.md
-[blockchain]: https://exonum.com/doc/api/java-binding-core/0.4/com/exonum/binding/blockchain/Blockchain.html
-[build-description]: https://github.com/exonum/exonum-java-binding/blob/v0.4/exonum-java-binding/service-archetype/src/main/resources/archetype-resources/pom.xml
+[abstractservice]: https://exonum.com/doc/api/java-binding-core/0.5/com/exonum/binding/service/AbstractService.html
```

Comment:
I think it makes sense to use the full version everywhere since 0.5.0:
- for API docs (0.5.0)
- for tags (including the project specifier, i.e., `ejb/v0.5.0`)

Reply:
Ok. I noticed that in checklist we have 0.5.0 but all previous releases didn't have patch version. Why the change?

Reply:
To bring consistency, ensure the scheme in use works for any use case (e.g., patch releases that might change documentation) and reduce the likelihood of errors when dealing with versions in various URLs.